pandas/core/resample.py

```diff
@@ -1627,18 +1629,21 @@ def _get_timestamp_range_edges(first, last, offset, closed='left', base=0):
     -------
     A tuple of length 2, containing the adjusted pd.Timestamp objects.
     """
-    if not all(isinstance(obj, pd.Timestamp) for obj in [first, last]):
```

Comment:
This is not needed AFAICT. All internal calls to this method have Timestamp inputs.